README.md

```diff
@@ -179,7 +179,7 @@ The matrix type can be explicitly set with `set_matrixtype!(ps, key)` where the
 
 | enum                 | integer | Matrix type                               |
 |--------------------- |---------| ----------------------------------------  |
-| REAL_SYM             | 1       | real and structurally symmetric           |
+| REAL_STRUCT_SYM      | 1       | real and structurally symmetric           |
```

Comment:
While the new name is better, it is a bit breaking to change it, can we keep the `REAL_SYM` as an undocumented enum instance and swap to `REAL_STRUCT_SYM` if it used?

Reply:
Of course, whatever you think is best! I'm not sure how to do that though... Could you hold my hand a little bit and clarify what you mean by "swap" here?

Reply:
Sorry to ping you here as well, but I'm not sure what to do with this one (not sure what you meant by "swapping"). Of course, I'm happy to just revert to the old `REAL_SYM` name and leave the name-change to a separate PR. What do you think?

Reply:
OK I've reverted the name change. It can be changed in a different PR if someone else feels strongly about it I guess.